doc/sphinx/language/core/inductive.rst

```diff
@@ -247,7 +247,7 @@ the same parameter values of its specification.
      because the conclusion of the type of constructors should be :g:`listw A`
      in both cases.
 
-   + A parameterized inductive definition can be defined using annotations
+   + A parameterized inductive definition can be defined using indices
```

Comment:
What about the uses of "annotation" on lines 420, 423, 445 and 446?

Reply:
Those are about the `{struct _}` on fixpoints, I don't think they have a name other than annotation.

"Universe annotation" could be "universe instance" instead or maybe would need to be "explicit universe instance".